exla/mix.exs

```diff
@@ -26,9 +26,20 @@ defmodule EXLA.MixProject do
         "compile.extract_xla": &extract_xla/1,
         "compile.cached_make": &cached_make/1
       ],
-      make_env: %{
-        "MIX_BUILD_EMBEDDED" => "#{Mix.Project.config()[:build_embedded]}"
-      }
+      make_env: fn ->
+        cwd_relative_to_priv =
+          Mix.Project.app_path()
+          |> Path.join("priv")
+          |> Path.relative_to_cwd()
```

Comment:
This assumes the priv dir is within the current working directory, but it should always be the case, since it's always in `_build`?

Reply:
You can have the build anywhere by using the `build_path` mix.exs configuration :(

Reply:
Perhaps `cd`-ing into `build_path` before calculating `Path.relative_to_cwd` can resolve this?

Reply:
We need to compute the relative path to the deps folder, which can be anywhere in the system, and the build folder, which can also be anywhere. Perhaps what we have to do is to call realpath and, depending if it return status 0 or 1, we act accordingly.

Reply:
I pushed an alternative version that manually computes the relative path. Let me know if that works, otherwise using `realpath` with status check sounds fine.